jenkins_epo/extensions/core.py

```diff
@@ -363,6 +363,22 @@ def run(self):
         ))
 
 
+class SecurityExtension(Extension):
+    stage = '00'
+
+    def begin(self):
```

Comment:
Est-ce-que c'est mieux en utilisant author ?

```

    def begin(self):
        author = getattr(self.current.head, 'author', None)

        if not author:
            # In case the extension is running on a branch
            return
```